common/persistence/elasticsearch/client/client.go

```diff
@@ -56,6 +58,13 @@ type (
 		GetMapping(ctx context.Context, index string) (map[string]string, error)
 	}
 
+	// Combine ClientV7 with Client interface after ES v6 support removal.
+	ClientV7 interface {
+		OpenPointInTime(ctx context.Context, index string, keepAliveInterval string) (string, error)
```

Comment:
`PointInTime` -> `Instant` or `Time`?

Reply:
It's really a snapshot, isn't it? There are three things here: (1) a snapshot that we're creating at (2) some point-in-time and (3) identified by an id. So it strike me that the API should be CreateSnapshot():SnapshotID and DeleteSnapshot(SnapshotID). The point-in-time is implicit in the semantics of a snapshot.

Naming is hard.

Reply:
I am trying to use names from Elasticsearch itself here.

Reply:
Fair enough - I thought the point was to be an abstraction of ES but if we're mapping directly to ES primitives that PIT makes sense.